docs/ref-manual/cvl/rules.md

```diff
@@ -25,23 +34,133 @@ params ::= cvl_type [ id ] { "," cvl_type [ id ] }
 Overview
```

Comment:
do we have a placeholder for cvl_type? might be worth to add a link from the grammar

Reply:
I have a separate PR where I add links for all the productions.  I think it's already merged into the master branch

Reply:
https://docs.certora.com/en/latest/docs/ref-manual/cvl/rules.html?highlight=cvl_type#syntax